Language/Structure/Control Structure/switchCase.adoc

```diff
@@ -83,6 +82,29 @@ switch (var) {
 ----
 [%hardbreaks]
 
+
+[float]
+=== Notes and Warnings
+// Add useful notes, tips, caveat, known issues, and warnings about this Reference term
+If you need to declare a variable in a case, use curly brackets {} to define a scope within the case statement to avoid a "crosses initialization of" compiler error. 
+[source,arduino]
+----
+switch (var) {
+  case 1:
+    //do something when var equals 1
+    break;
+  case 2: {
+    //do something when var equals 2
+    int var2 = 1337; //to declare a variable you need the curly brackets
+    break;
+  }
+  default:
+    // if nothing else matches, do the default
+    // default is optional
```

Comment:
```suggestion
```
This information does not add any value to the demonstration of the use of braces to create a scope in a case and has already been mentioned in the existing general example sketch.